monitor/widgets/src/main/java/org/datacleaner/monitor/scheduling/widgets/SchedulePanel.java

```diff
@@ -70,19 +77,25 @@
     Button executeButton;
 
     @UiField
+    InlineLabel latestExecutionLabel;
+    
```

Comment:
When I reformat the code in my IDE the whitespaces from this "empty" line are removed.